mms/service.py

```diff
@@ -112,7 +112,7 @@ def predict(self, batch):
                            self.context.model_name, len(input_batch), len(ret))
             return create_predict_response(None, req_id_map, "number of batch response mismatched", 503)
 
-        duration = int((time.time() - start_time) * 1000)
+        duration = round((time.time() - start_time) * 1000, 2)
```

Comment:
If we are adding individual metrics, do we still need this ```PREDICTION_METRIC```? This seems to have the "preprocess" "postprocess" and "inference" times as well.

Reply:
Predictions metrics is in our code base, this will get added no matter what custom service code do.

pre/infer/post are only in the template

Reply:
I agree. Customers code might not have it.